requirements.txt

```diff
@@ -7,8 +7,8 @@ click==7.1.2
 Django==3.0.10
 django-environ==0.4.5
 django-extensions==3.0.9
+django-recurrence==1.10.3
 djangorestframework==3.11.1
-django-environ==0.4.5
```

Comment:
There were some duplicate things in requirements, so these removals are from that. Probably from some merge conflict or something where pip freeze wasn't used.